examples/ui/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func ExampleHandler1(w http.ResponseWriter, r *http.Request) {
+	htmlPage := "<!doctype html><html lang=en><head><meta charset=utf-8><title>blah</title></head><body><p class='example-paragraph'>Example web page</p></body></html>"
+	fmt.Fprint(w, htmlPage)
```

Comment:
Fprint returns two arguments right? The second argument is a possible error. Aware this is an example so not a must change comment, and can also see in other example tests (from before this PR) there are errors missed too. This is just me 2 cents.